soc/nxp/lpc/lpc54xxx/Kconfig.defconfig

```diff
@@ -13,6 +13,7 @@ config NUM_IRQS
 	default 40
 
 config MCUX_CORE_SUFFIX
```

Comment:
All `MCUX_CORE_SUFFIX` usage needs to move over to `Kconfig` to align with other SoCs